documentation/API-design-guidelines.md

```diff
@@ -1973,6 +1973,11 @@ Note1: This enumeration is also defined in `event-subscription-template.yaml` (p
 
 Note2: The "subscription-ends" notification is not counted in the `subscriptionMaxEvents`. (for example, if a client request set `subscriptionMaxEvents` to 2, and later, received 2 notifications, then a third notification will be sent for "subscription-ends").
 
+Note3: In the case of ACCESS_TOKEN_EXPIRED termination reason sending the notification once the token expired is useless. To avoid this case, following rules are defined :
+
+- For explicit subscription, implementation should send ACCESS_TOKEN_EXPIRED termination event just before the token expiration date (the 'just before' value is at the hands of each implementation). The following sentence must be added for the `accessTokenExpiresUtc` attribute documentation: "An absolute UTC instant at which the token shall be considered expired. In the case of ACCESS_TOKEN_EXPIRED termination reason, implementation should notified client before the expiration date."
+- For implicit subscription following sentence must be added for the `accessTokenExpiresUtc` attribute documentation: "An absolute UTC instant at which the token shall be considered expired. Token expiration should happen after the requested _resource_ , for the client to be notified of any change during the _resource_. If the token expires while the _resource_ is still active, the client will stop receiving notifications.". The _resource_ word must be replaced by the entity managed by the subscription (session, payment, etc.).
```

Comment:
```suggestion
- For implicit subscription following sentence must be added for the `accessTokenExpiresUtc` attribute documentation: "An absolute (UTC) timestamp at which the token shall be considered expired. Token expiration should occur after the expiration of the requested _resource_, allowing the client to be notified of any changes during the _resource_'s existence. If the token expires while the _resource_ is still active, the client will stop receiving notifications.". The _resource_ word must be replaced by the entity managed by the subscription (session, payment, etc.).
```

Reply:
@bigludo7 Some suggestions prepared with the help of AI :)

Reply:
LGTM with @rartych suggestions